glossary.md

```diff
@@ -39,3 +39,21 @@ USA -- typeOf ------------> Country
 USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
+
+- **[Measurement method](https://datacommons.org/browser/measurementMethod)**
```

Comment:
How about adding the link separately in the body (so we can describe that it's a link to the node definition).

Reply:
I guess I am not sure what is meant by this. For most of the definitions here, the link is included as part of the header. I suppose we could change that for all of them, or we could add an additional link in the definition body--further clarification requested!